stdlib/system/src/main/ballerina/src/system/system_errors.bal

```diff
@@ -47,5 +47,11 @@ public const FILE_NOT_FOUND_ERROR = "{ballerina/system}FileNotFoundError";
 # Represents an `FileNotFoundError` with a detailed message.
 public type FileNotFoundError error<FILE_NOT_FOUND_ERROR, Detail>;
 
+# Represents the error code for process execute error.
+public const PROCESS_EXEC_ERROR = "{ballerina/system}ProcessExecError";
+
+# Represents an `ProcessExecError` with a detailed message.
+public type ProcessExecError error<PROCESS_EXEC_ERROR, Detail>;
+
 # Represents System related errors.
-public type Error InvalidOperationError|PermissionError|FileSystemError|FileNotFoundError;
+public type Error InvalidOperationError|PermissionError|FileSystemError|FileNotFoundError|ProcessExecError;
```

Comment:
Isn't this backward incompatible?

Reply:
This error type is something we missed in previous releases. I think it is better to correct this now. There is an issue if the users handled it at a low level.

Reply:
Or we will reuse existing error type for patch release and correct it in minor release

Reply:
Yeah, +1. We'll still have the details right? Only the reason will be incorrect.

Reply:
Yes. fixed in 0e7ea63